submissions/ses3332002/friends-app/app.js

```diff
@@ -6,51 +6,55 @@ const ageFromOption = document.querySelector("#age_from");
 const ageToOption = document.querySelector("#age_to");
 const searchNameOption = document.querySelector(".filter_name");
 const resetButton = document.querySelector(".reset_button");
+const renewButton = document.querySelector(".renew_button");
 const requestString = "https://randomuser.me/api/?nat=gb,us,fr&results=60&inc=gender,name,location,email,picture,dob";
 let receivedData = [];
-let dataToRender = [];
 let filterCriterionSex;
 let filterCriterionAge
 let filterCriterionName;
 let sortCriterion;
 hamburgerMenu.addEventListener("click", () => hamburgerMenu.classList.toggle("hamburger_menu-opened"));
-resetButton.addEventListener("click", resetApp);
+resetButton.addEventListener("click", prepareForm);
+renewButton.addEventListener("click", prepareList);
 optionForm.addEventListener("input", changeData);
-resetApp();
+prepareList();
   
-function resetApp() {
+function prepareForm() {
   filterCriterionSex = () => true;
   filterCriterionAge = () => true;
   filterCriterionName = () => true;
   sortCriterion = () => false;
   optionForm.reset();
-  
+  render(receivedData.slice(), sortCriterion, filterCriterionSex, filterCriterionAge, filterCriterionName);
+};
```

Comment:
Use a relevant function name in relation to what it does

Reply:
I remade it